File-Formats/DocIO/Create-Word-document-in-Linux.md

```diff
@@ -24,7 +24,7 @@ dotnet new console
 
 {% endtabs %}
 
-![Create .NET Core console application on Linux](Linux-images/CreateNewProject1.png)
+![Create .NET Core console application on Linux](linux-images/createnewproject.png)
```

Comment:
give hypen for each word, 
example:
create-new-project-on-linux

Update the changes in all needed place.

Note: 
Given image name should be meaningful.
If you change the image name in MD file, you should change the image name in corresponding image folder.

Reply:
Comments resolved